src/meili/routing.cc

```diff
@@ -162,9 +162,11 @@ inline bool IsEdgeAllowed(const baldr::DirectedEdge* edge,
                           const sif::cost_ptr_t& costing,
                           const Label& pred_edgelabel,
                           const baldr::GraphTile* tile) {
+  bool i_dont_care_about_time_restrictions_here = false;
```

Comment:
This could be an interesting TODO actually. Because you can specify timestamps with the map matching. We currently use it to only exclude paths which would be too far for the amount of time passed, but we could use the epoch time at the location to adhere to restrictions. This might be contentious though as some use-cases might want the map match to ignore restrictions and some might want them to adhere to them. anyway, it might be nice to put a note here?